From fe0f221167cffd6fdea67071bd7b13f3abcee93b Mon Sep 17 00:00:00 2001
From: q66 <daniel@octaforge.org>
Date: Thu, 26 Dec 2019 22:24:13 +0100
Subject: [PATCH] Remove libebl linkage

Since a recent version of elfutils libebl is a part of libdw, so
this is necessary to fix build.
---
 Makefile | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git Makefile Makefile
index 1fdb628..df21b93 100644
--- Makefile
+++ Makefile
@@ -50,7 +50,7 @@ OBJ_PART=$(patsubst %.c,%.o,$(SRC_PART))
 SRC_ARCH = arch/arm.c arch/arm64.c arch/x86.c arch/x86_64.c arch/ia64.c arch/ppc64.c arch/s390x.c arch/ppc.c arch/sparc64.c
 OBJ_ARCH=$(patsubst %.c,%.o,$(SRC_ARCH))
 
-LIBS = -ldw -lbz2 -lebl -ldl -lelf -lz
+LIBS = -ldw -lbz2 -ldl -lelf -lz
 ifneq ($(LINKTYPE), dynamic)
 LIBS := -static $(LIBS)
 endif
-- 
2.24.0

